### ISSUE TITLE
Improve the UX oracle issues

### ISSUE THREAD

**Issue by Andrew718PLTS** [mento-protocol/mento-web#156]
## Description
The scope is to improve a UX for the "no valid median", "trading is suspended", "overflow x1y1", "can't create fixidity number larger than" error cases. Currently, we have only error notification toast with the "Unable to fetch swap out amount" message that is not so responsive - I guess we have to define a nice error toast with more responsive message that describe the certain case
(e.g: "An exchange price for <token> currently unavailable - please try again later").

## Acceptance Criteria
- [x] An error toast with a specified message appears when there are described errors

**Comment by philbow61** [comment]
should be fixed together with https://app.zenhub.com/workspaces/planning-65b8ace7a595330035ca810b/issues/gh/mento-protocol/mento-web/158

**Comment by chapati23** [comment]
@andrew.polovets @philip.raetsch i'm confused, can we close this ticket or not?

**Comment by Andrew718PLTS** [comment]
@chapati23 No, it's actually in progress (I should fix some things that were left @nvtaveras) - here is the [PR](https://github.com/mento-protocol/mento-web/pull/182).

In Zenhub, it looks like this only because I can't re-link my new PR that I mentioned here in the Zenhub board (Zenhub links once someone mentioned PR (fixes: #1) and can't link with the next mentions)
<img width="375" alt="Image" src="https://github.com/user-attachments/assets/bb0054cf-3446-4aed-a447-9ca3fd4ae526" />
